_posts/2020-02-22-gsoc2020.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: Google Summer of Code 2020
+---
+
+<p>
+<img
+src="https://developers.google.com/open-source/gsoc/images/gsoc2016-sun-373x373.png"
+title="Google Summer of Code 2020" alt="Google Summer of Code 2020"
+style="float: left; height: 4.5em; " />
+</p>
+
+MDAnalysis has been accepted as an [organization][org] into [Google Summer of Code 2020][gsoc]!
+If you are a student who is interested in working with us
+this summer, please read the advice and links below and write to us on the
+[mailing list].
+
+We are looking forward to all applications from interested students
+(*undergraduate* and *postgraduate*).
+
+The application window **deadline** is **March 31, 2020 at 18:00 (UTC)**. As
+part of the application process you must familiarize yourself with [Google
+Summer of Code 2020][gsoc]. 
+
+[Apply] as soon as possible; the application window opens on March 16,
+2020 but you should start engaging with us sooner than that: read on.
```

Comment:
Should engage with us as soon as possible.

Reply:
We should mention that early and continuously engaging with us is the best way to improve chances for a student. Especially since our main goal is to attract new core developers.